src/Utils/removeDuplicates.ts

```diff
@@ -0,0 +1,50 @@
+export const csvGroupByColumn = (
+  data: string,
+  groupByColumn: string,
+  delimiter = ";",
+): string => {
+  const [header, ...datalines] = data.trim().split("\n"); // Split the data into individual lines
+  const headerColumns = header.split(","); // Extract the header columns
```

Comment:
code is self explanatory

```suggestion
  const [header, ...datalines] = data.trim().split("\n");
  const headerColumns = header.split(",");
```